docs/source/conf.py

```diff
@@ -60,6 +60,7 @@
     ("py:class", "Session"),
     ("py:exc", "nbconvert.preprocessors.CellExecutionError"),
     ("py:class", "nbdime.diff_format.DiffEntry"),
+    ("py:class", "nbdime.diffing.config.DiffConfig"),
```

Comment:
I am not sure about this one, but it apparently satisfies Sphinx/RTD. Please provide guidance if it needs to be resolved differently.

Reply:
> You need to add it to the `nitpick_ignore`.

All right, thank you. I just figured it out already.

Reply:
could add nbdime's url to the intersphinx mapping, but not necessary here